.github/workflows/main.yml

```diff
@@ -174,7 +174,8 @@ jobs:
         python -m pip install wheel setuptools mypy
     - name: Run mypy
       run: |
-        cd python-package && mypy . || true
+        cd python-package
+        mypy ./xgboost/dask.py ../tests/python/test_with_dask.py --follow-imports=silent
```

Comment:
Can we still run mypy on other Python files? (We can keep it nonblocking)

Reply:
Yes.  Let me do that.